backend/invoke_ai_web_server.py

```diff
@@ -637,6 +636,25 @@ def image_progress(sample, step):
                             "height": height,
                         },
                     )
+
+                if generation_parameters['progress_latents']:
+                    image = self.generate.sample_to_lowres_estimated_image(sample)
+                    (width, height) = image.size
+                    buffered = io.BytesIO()
+                    image.save(buffered, format="PNG")
+                    img_base64 = "data:image/jpeg;base64," + base64.b64encode(buffered.getvalue()).decode('UTF-8')
```

Comment:
this has `format="PNG"` but then it's given the `image/jpeg` mime type.

Reply:
so it doooes. spot the non-web programmer (me)